dataactvalidator/filestreaming/loadFile.py

```diff
@@ -10,15 +11,39 @@ def loadCgac(filename):
 def loadObjectClass(filename):
     LoaderUtils.loadCsv(filename,ObjectClass,ValidatorValidationInterface(),{"max_oc_code":"object_class_code","max_object_class_name":"object_class_name"},{"object_class_code":{"skip_duplicates":True}})
 
+def loadSF133(filename,skipClear = False):
+    """ Load SF133 files, set skipClear to True for second file """
+    LoaderUtils.loadCsv(filename,SF133,ValidatorValidationInterface(),{"ata":"allocationtransferagencyidentifier","aid":"agencyidentifier","availability_type_code":"availabilitytypecode","bpoa":"beginningperiodofavailability","epoa":"endingperiodofavailability","main_account":"mainaccountcode","sub_account":"subaccountcode","fiscal_year":"fiscal_year","period":"period","line_num":"line","amount_summed":"amount"},{"allocationtransferagencyidentifier":{"pad_to_length":3},"agencyidentifier":{"pad_to_length":3},"mainaccountcode":{"pad_to_length":4},"subaccountcode":{"pad_to_length":3}},skipClear=skipClear)
+
 def loadProgramActivity(filename):
     LoaderUtils.loadCsv(filename, ProgramActivity, ValidatorValidationInterface(), {"year":"budget_year","agency_id":"agency_id",
         "alloc_id":"allocation_transfer_id","account":"account_number","pa_code":"program_activity_code","pa_name":"program_activity_name"},
         {"program_activity_code":{"pad_to_length":4},"agency_id":{"pad_to_length":3},"account_number":{"pad_to_length":4},"allocation_transfer_id":{"pad_to_length":3}})
 
-def loadDomainValues(basePath):
+
+def loadDomainValues(basePath, localSFPath = None):
+    """ Load all domain value files, localSFPath is used to point to a SF-133 file, if not provided it will be downloaded from S3  """
     loadCgac(os.path.join(basePath,"cgac.csv"))
     loadObjectClass(os.path.join(basePath,"object_class.csv"))
+
+    # SF 133 is kept on S3, so need to download that
+    reader = CsvS3Reader()
+
+    if localSFPath is not None:
+        # Load SF 133 from same path
+        loadSF133(localSFPath)
+    else:
+        # Download files if using aws, if not they will need to already be in broker_files location
+        if(CONFIG_BROKER["use_aws"]):
+            print("Pulling values from config: " + str(CONFIG_BROKER["aws_region"]) + " " + str(CONFIG_BROKER["aws_bucket"]) + " " + str(CONFIG_BROKER["sf_133_folder"]) + " " + str(CONFIG_BROKER["sf_133_file_one"]))
```

Comment:
Did you want to leave this print statement in (not sure how terrible it is to print out the AWS info from the config file)?